ivy_tests/test_ivy/test_frontends/test_jax/test_nn/test_non_linear_activations.py

```diff
@@ -9,7 +9,7 @@
 @handle_frontend_test(
     fn_tree="jax.nn.relu",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float_and_integer"),
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
```

Comment:
I think `numeric` covers both of these (otherwise it won't generate integers any more)